src/views/Project/index.jsx

```diff
@@ -111,14 +141,21 @@ export default class Project extends Component {
       [];
 
     return (
-      <Fragment>
-        <header>
-          <Typography variant="display2" align="center">
+      <div className={classes.root}>
+        <AppBar position="absolute" className={classes.header}>
+          <IconButton className={classes.link} component={NavLink} to="/">
+            <ArrowLeftIcon />
+          </IconButton>
+          <Typography
+            className={classes.title}
+            variant="display1"
+            align="center"
+            justifyContent="center"
```

Comment:
Is `justifyContent` a valid prop?

Reply:
it isn't! sorry i played with it and forgot to remove it ><